source/optional_features/delayed capture

```diff
@@ -0,0 +1,12 @@
+#Delayed capture 
+
+GOV.UK Pay supports delayed capture of payments.
+
+To use delayed capture, you can include `"delayed_capture": true` in the body of a request to create a payment. 
+
+The user experience matches the current payment flow. Once the user selects “Confirm” on the “Confirm your details” page, a request can be made by your service to capture the payment. 
+To do this, your service calls the POST /v1/payments/{paymentId}/capture endpoint. A 204 response indicates success; a 409 response indicates that the payment is not in an appropriate state to be captured. 
+
+You must make the capture request within 90 minutes of when the payment is created (regardless of how long the user takes to complete the payment flow) or it will expire.
+
+A payment receipt email is sent to the user once the capture request is received and accepted. 
```

Comment:
Who sends the payment receipt email? The service or GOV.UK Pay?

Reply:
If your service has enabled payment receipt emails, GOV.UK Pay will send a payment receipt email to the user once your service's capture request is received and accepted.